c/CLU.clu

```diff
@@ -0,0 +1,4 @@
+start_up = proc ()
+ po: stream := stream$primary_output ()
+ stream$putl (po, "Hello World")
```

Comment:
Should these lines have a space of indent?

Reply:
Lines 2 and 3 should have indentation, not necessarily only one space, sorry for that one.